### ISSUE TITLE
Missing field under "Provider Presets" when compared to that on the Create Cluster screen

### ISSUE THREAD

**Issue by archups** [kubermatic/kubermatic#9726]
### What happened?

From UI. 
"Provider Presets" field options have a mismatch.
    --> Folder name field is not available as part of the vSphere Provider field at the "Settings" step under "Admin Panel" > "Provider Presets" when Provider is selected as vSphere. While it is available on the "Create Cluster" screen as part of the "Settings" > "Advanced Settings".

Also, not available as part of the preset CRD when checked this documentation https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/administration/admin_panel/presets_management/

### Expected behavior

It would be nice to have consistency between Provider Preset fields and those available during cluster creation process. 
For "Provider Presets" under "Settings" tab the "Folder name" should be available for configuration. So that it can be auto-populated when preset is selected during Cluster creation process. 

Also, should be supported as part the preset CRD. 

### How to reproduce the issue?

Navigate to Admin Panel > Provider Presets, provide name at  "Preset" tab and move to next  tab "Provider", select vSphere as provider, then move next to "Settings"  tab. On this final tab, the "Folder Name" field is not available for configuration. 

### How is your environment configured?

- KKP version: 2.19.2
- Shared or separate master/seed clusters?: Separate Master/Seed Cluster

### Provide your KKP manifest here (if applicable)

<details>

```yaml
# paste manifest here
```

</details>

### What cloud provider are you running on?

vSphere

### What operating system are you running in your user cluster?

Ubuntu 20.04

**Comment by moadqassem** [comment]
@csengerszabo Why would the folder consider as a preset? @archups did this come from a customer? The folder is a cluster specific field, meaning, for each cluster that we create a new folder will be generated if the user didn't assign any folder name. Thus if users added a preset for this, it will be reflected on all user clusters which is IMHO isn't optimal nor organized. WDYT ?

**Comment by csengerszabo** [comment]
@moadqassem can you help in making clear what the folder setting actually does? What happens exactly when I assign a folder name during the cluster creation? More specifically:
- what is the goal of adding a folder to a provider preset?
- should it be an existing folder or I can specify a name and it will create one?
- where is that folder?
- what is that folder supposed to store?

What happens when I don't add a folder name during cluster creation?

**Comment by archups** [comment]
>  @archups did this come from a customer?
@csengerszabo  yes, this is from customer end.

**Comment by csengerszabo** [comment]
@archups I've got the information from @moadqassem that the folder setting for the presets should be done on the seed and [the folder can be set per datacenter](https://github.com/kubermatic/kubermatic/blob/9f44a5311ff6c0a46f3ef04e8154d582c4f77668/pkg/apis/kubermatic/v1/datacenter.go#L506)

**Comment by archups** [comment]
@csengerszabo screenshot for reference to missing "folder" field under Provider Preset. No "Folder" field option available in preset screen to configure the value per datacenter for vSphere here.
<img width="366" alt="image" src="https://user-images.githubusercontent.com/25936785/178676852-3aa25395-aa50-4fa2-ba98-f9979a055f66.png">

Screenshot of the create cluster "Settings" screen which has "Folder" as part of advanced configuration.  
<img width="948" alt="image" src="https://user-images.githubusercontent.com/25936785/178677040-3709d043-5074-449d-893e-5af62235170a.png">

**Comment by archups** [comment]
@csengerszabo , As discussed with @moadqassem - understanding is that folder (rootpath) and other settings (such as datastore, etc.) are taken care as part of seed object definition for each of the vSphere datacenter. However, from UI side, we do see the possibility on the screen that preset can capture the datacenter specific settings (such as datastore, etc.) . Hence to have consistency, it would be good to rethink on the addition of "Folder" field on the Provider Preset "Create Preset" screen as we are already allowing  other datacenter specific settings  (such as datastore, etc.)  to get captured.

**Comment by ahmadhamzh** [comment]
blocked by https://app.zenhub.com/workspaces/work-in-progress---the-a-team-5fa971db111c0d0018ce0008/issues/kubermatic/kubermatic/11011
